developers/blobstream.md

```diff
@@ -14,128 +16,154 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
+Integrating your L2 with Blobstream X requires two components: your onchain smart
 contract logic, and your offchain client logic. The next three sections cover these
 topics:
 
-- [Integrate with Blobstream contracts](./blobstream-contracts.md)
-- [Integrate with Blobstream client](./blobstream-offchain.md)
-- [Querying the Blobstream proofs](./blobstream-proof-queries.md)
+- [Integrate with Blobstream X contracts](./blobstreamx-contracts.md)
+- [Integrate with Blobstream X client](./blobstreamx-offchain.md)
+- [Querying the Blobstream X proofs](./blobstreamx-proof-queries.md)
 
 ### Deployed contracts
 
-You can interact with the Blobstream contracts today on testnet. The Blobstream Solidity
-smart contracts are currently deployed on the following Ethereum testnets:
+You can interact with the Blobstream X contracts today on testnet. The
+Blobstream X Solidity smart contracts are currently deployed on
+the following Ethereum testnets:
 
 <!-- markdownlint-disable MD013 -->
 
-| Contract     | EVM network      | Contract address                                                                                                                | Attested data |
+| Contract     | EVM network      | Contract address                                                                                                                | Attested data on Celestia |
 | ------------ | ---------------- | ------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| Blobstream   | Sepolia          | [`0x3a5cbB6EF4756DA0b3f6DAE7aB6430fD8c46d247`](https://sepolia.etherscan.io/address/0x3a5cbB6EF4756DA0b3f6DAE7aB6430fD8c46d247) | Mocha testnet |
-| Blobstream   | Arbitrum Sepolia | [`0x040769edbca5218e616c8eb16e4faea49ced5e33`](https://sepolia.arbiscan.io/address/0x040769edbca5218e616c8eb16e4faea49ced5e33)  | Mocha testnet |
-| Blobstream X | Goerli           | [`0x67ea962864cdad3f2202118dc6f65ff510f7bb4d`](https://goerli.etherscan.io/address/0x67ea962864cdad3f2202118dc6f65ff510f7bb4d)  | Mocha testnet |
+| Blobstream X  | Ethereum Mainnet          | [`Not yet deployed`](https://etherscan.io/address/0xTODO) | [Mainnet Beta](../nodes/mainnet.md) |
+| Blobstream X | Arbitrum One | [`Not yet deployed`](https://arbiscan.io/address/0xTODO)  | [Mainnet Beta](../nodes/mainnet.md) |
+| Blobstream X | Base           | [`Not yet deployed`](https://goerli.etherscan.io/address/0xTODO)  | [Mainnet Beta](../nodes/mainnet.md) |
+| Blobstream X | Ethereum Sepolia           | [`Not yet deployed`](https://sepolia.etherscan.io/address/0xTODO)  | [Mocha testnet](../nodes/mocha-testnet.md) |
+| Blobstream X | Arbitrum Sepolia           | [`Not yet deployed`](https://sepolia.arbiscan.io/address/0xTODO)  | [Mocha testnet](../nodes/mocha-testnet.md) |
 
 <!-- markdownlint-enable MD013 -->
 
-## Blobstream vs. data availability committees (DACs)
+## Blobstream X vs. data availability committees (DACs)
```

Comment:
```suggestion
## Blobstream vs. data availability committees (DACs)
```

This should stay as Blobstream because we are discussing the concept of Blobstream, not the implementation